<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogWindowWidget</class>
 <widget class="QWidget" name="dialogWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#dialogWindowWidget{
	border: 0px;	
	background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <widget class="QFrame" name="styleFrame">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Comic Sans MS&quot;;
	border: 2px solid 	rgb(132, 93, 194)	 ;
	background-color: 	rgb(250, 179, 24)	;
	color: rgb(245, 246, 250);
	border-radius: 7px;
}
QPushButton:hover{
	background-color: rgb(0, 201, 183)				;}
QPushButton:pressed{
		background-color: rgb(0, 193, 174)		;}
QLabel{
	font: 10pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 7px;	
	background-color: rgba(255, 255, 255, 0);
}
#dialogheader{
	font: 14pt &quot;Comic Sans MS&quot;;
	border-radius: 7px;
}
#dialogheaderFrame{
	background-color:rgb(132, 93, 194);
border: 0px;
	border-bottom-right-radius:7px;
}
#styleFrame{
background-color:rgb(250, 179, 24);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="dialogText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>пу пук пукпу  пупу.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QFrame" name="dialogheaderFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="dialogheader">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Название окна</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="dialogOkBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dialogCancelBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
